### PR TITLE
Add Twitter-Link to Homepage (#131)

### PR DESCRIPTION
**Pull request by bjoernma**

### CODE REVIEW

**Comment by Nakaner** [comment]
@bjoernma I removed the "Contact" because it is obvious from the titles "email" and "Twitter" that they are contact channels. In addition, I fixed CSS-related errors.

File: mobile.php [hunk 1]
```diff
@@ -126,11 +126,12 @@
 			<div class="box">
 				<a style="display:block" class="resultEntry" id="infoButton" href="https://wiki.openstreetmap.org/wiki/OpenRailwayMap"><?=_("More Info")?></a>
 				<a style="display:block" class="resultEntry" id="infoButton" href="https://blog.openrailwaymap.org/"><?=_("Blog")?></a>
-				<a style="display:block" class="resultEntry" id="contactButton" href="#"><?=_("Contact")?></a>
 				<a style="display:block" class="resultEntry" id="paypalButton" href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&amp;hosted_button_id=9KCKT39N7AGL8"><?=_("Donate")?></a>
 				<a style="display:block" class="resultEntry" id="imprintButton" href="/imprint"><?=_("Imprint &amp; Privacy Policy")?></a>
 				<a style="display:block" class="resultEntry" id="permalinkButton" href=""><?=_("Permalink")?></a>
 				<a style="display:block" class="resultEntry" id="desktopButton" href=""><?=_("Desktop version")?></a>
+                                <a style="display:block" class="resultEntry" id="contactButton" href="#"><?=_("Email")?></a>
```
Comment (DerDakon) [inline mobile.php:133]: The indentation is different.

**Comment by DerDakon** [comment]
> I removed the "Contact" because it is obvious from the titles "email" and "Twitter" that they are contact channels.

You left it in for canvas.php, was that intentional?

**Review by DerDakon — APPROVED** [review]
I have changed ```id="twitterbutton"``` to ```id="twitterButton"``` to match the other buttons and wrapped the line in index.php.

**Pull request merged**